.travis.yml

```diff
@@ -1,48 +1,39 @@
 language: python
 
-
 matrix:
   include:
     - env: TOXENV=docs
     - env: TOXENV=pep8
     - env: TOXENV=py3pep8
+    - env: TOXENV=isort
     - env: TOXENV=packaging
-    - python: 2.7
-      env: TOXENV=py27
-    - python: 3.3
-      env: TOXENV=py33
-    - python: 3.4
-      env: TOXENV=py34
-    - python: 3.5
-      env: TOXENV=py35
-    - python: 3.6
-      env: TOXENV=py36
-    - python: nightly
-      env: TOXENV=py37
-    - python: pypy
-      env: TOXENV=pypy
-    - python: 2.7
-      env: TOXENV=py27 VENDOR=no WHEELS=yes
-    - python: 3.6
-      env: TOXENV=py36 VENDOR=no WHEELS=yes
-
+    - env: TOXENV=py27
+      python: 2.7
```

Comment:
I did it to make the keys sorted, alphabetically. Aesthetic change, not functional.